### PR TITLE
Refactored SQL Observability DB/Cloud console pages to be shared.

### PR DESCRIPTION
**Pull request by ianjevans**
This PR pulls the Sessions, Statements, and Transactions page content into a common include with Liquid conditionals to handle different links and UI elements.

https://cockroachlabs.atlassian.net/browse/CRDB-2505

### CODE REVIEW

**Comment by stbof** [comment]
The Statement Details page for Cloud isn't correct. It doesn't have any of the tabs shown here: https://deploy-preview-12263--cockroachdb-docs.netlify.app/docs/cockroachcloud/statements-page.html#activate-diagnostics-collection-and-download-bundles

**Comment by ianjevans** [comment]
This is what I see after, but you have to have had statistics collected for any of the additional details to show up.

Screen Shot 2021-11-12 at 9 29 50 AM

**Comment by ianjevans** [comment]
But the Diagnostics tab is disabled for now on Serverless clusters, so I'll enclose it with a conditional.

File: _includes/common/ui/statements-page.md [hunk 1]
```diff
@@ -120,14 +120,12 @@ Diagnostics | Activate and download [diagnostics](#diagnostics) for this fingerp
 Click a SQL statement fingerprint to open **Statement Details**. For each statement fingerprint, the details include:
 
 - [Overview](#overview)
+{% if page.cloud != true %}
```
Comment (stbof) [inline _includes/common/ui/statements-page.md:123]: Line 116 has to be conditionalized out of cloud.

File: _includes/common/ui/statements-page.md [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+Retries | Cumulative number of [retries]({{ link_prefix }}transactions.html#transaction-retries) of statements with this fingerprint within the last hour.
+% of All Runtime  | How much time this statement fingerprint took to execute compared to all other statements that were executed within the time period. It is expressed as a percentage. The runtime is the mean execution latency multiplied by the execution count.
+Regions/Nodes | The regions and nodes on which statements with this fingerprint executed.
+Diagnostics | Activate and download [diagnostics](#diagnostics) for this fingerprint. To activate, click the **Activate** button. The column displays the status of diagnostics collection (`WAITING`, `READY`, OR `ERROR`). When the status is `READY`, click **Download bundle** to download the most recent diagnostics bundle. <br><br>Statements are periodically cleared from the Statements page based on the start time. To access the full history of diagnostics for the fingerprint, see the [Diagnostics](#diagnostics) section of the Statement Details page. <br><br>**Diagnostics** is not currently enabled for {{ site.data.products.serverless }} clusters.
```
Comment (stbof) [inline _includes/common/ui/statements-page.md:116]:
```suggestion
Diagnostics | Activate and download [diagnostics](#diagnostics) for this fingerprint. To activate, click the **Activate** button. The column displays the status of diagnostics collection (`WAITING`, `READY`, OR `ERROR`). When the status is `READY`, click **Download bundle** to download the most recent diagnostics bundle. <br><br>Statements are periodically cleared from the Statements page based on the start time. To access the full history of diagnostics for the fingerprint, see the [Diagnostics](#diagnostics) section of the Statement Details page. <br><br>**Diagnostics** is not enabled for {{ site.data.products.serverless }} clusters.
```

File: _includes/common/ui/sessions-page.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+Use the **Sessions** list to see the open sessions in the cluster. This includes active and idle sessions.
+
+{{site.data.alerts.callout_info}}
+A session is *active* if it has an open transaction (including implicit transactions, which are individual SQL statements), and *idle* if it has no open transaction. Active sessions consume hardware resources.
+{{site.data.alerts.end}}
+
+- If a session is active, the most recent SQL statement is displayed in the **Statement** column.
+- If a session is idle, **Transaction Duration**, **Statement Duration**, and **Statement** will display `N/A`.
+- To view [details of a session](#session-details), click the **Session Duration**.
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:19]:
```suggestion
- To view [details of a session](#session-details), click **Session Duration**.
```
```suggestion
- To view [details of a session](#session-details), click the **Session Duration**.
```

File: _includes/common/ui/sessions-page.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+{% endif %}
+
+- **Session** shows the ID of the connected session.
+	- **Session Start Time** shows the timestamp at which the session started.
+	- **Gateway Node** <a name="session-details-gateway-node"></a> shows the node ID and IP address/port of the [gateway]({{ link_prefix }}architecture/life-of-a-distributed-transaction.html#gateway) node handling the client connection.
+	- **Client Address** shows the IP address/port of the client that opened the session.
+	- **Memory Usage** shows the amount of memory currently allocated to this session, followed by the maximum amount of memory this session has ever allocated.
+	- The **Terminate Session** button ends the session. The client that holds this session will receive a "connection terminated" event.
+	- The **Terminate Statement** button ends the SQL statement. The session running this statement will receive an error.
+- **Transaction** will display the following information for an open transaction.
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:59]:
```suggestion
- **Transaction** displays the following information for an open transaction.
```
```suggestion
- **Transaction** will display the following information for an open transaction.
```

File: _includes/common/ui/sessions-page.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+	- **Transaction Start Time** shows the timestamp at which the transaction started.
+	- **Number of Statements Executed** shows the total number of SQL statements executed by the transaction.
+	- **Number of Retries** shows the total number of [retries]({{ link_prefix }}transactions.html#transaction-retries) for the transaction.
+	- **Number of Automatic Retries** shows the total number of [automatic retries]({{ link_prefix }}transactions.html#automatic-retries) run by CockroachDB for the transaction.
+	- **Priority** shows the [priority]({{ link_prefix }}transactions.html#transaction-priorities) for the transaction.
+	- **Read Only?** shows whether the transaction is read-only.
+	- **AS OF SYSTEM TIME?** shows whether the transaction uses [`AS OF SYSTEM TIME`]({{ link_prefix }}performance-best-practices-overview.html#use-as-of-system-time-to-decrease-conflicts-with-long-running-queries) to return historical data.
+	- **Memory Usage** shows the amount of memory currently allocated to this transaction, followed by the maximum amount of memory this transaction has ever allocated.
+- **Statement** will display the following information for an active statement.
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:68]:
```suggestion
- **Statement** displays the following information for an active statement.
```
```suggestion
- **Statement** will display the following information for an active statement.
```

File: _includes/common/ui/sessions-page.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+	- The SQL statement is shown.
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:69]:
```suggestion
	- The SQL statement.
```
```suggestion
	- The SQL statement is shown.
```

File: _includes/common/ui/sessions-page.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+	- **Execution Start Time** shows the timestamp at which the statement was run.
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:70]:
```suggestion
	- **Execution Start Time** the timestamp at which the statement was run.
```
```suggestion
	- **Execution Start Time** shows the timestamp at which the statement was run.
```

File: _includes/common/ui/sessions-page.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+	- **Distributed Execution?** shows whether the statement uses [Distributed SQL (DistSQL)]({{ link_prefix }}architecture/sql-layer.html#distsql) optimization.
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:71]:
```suggestion
	- **Distributed Execution?** whether the statement uses [Distributed SQL (DistSQL)]({{ link_prefix }}architecture/sql-layer.html#distsql) optimization.
```
```suggestion
	- **Distributed Execution?** shows whether the statement uses [Distributed SQL (DistSQL)]({{ link_prefix }}architecture/sql-layer.html#distsql) optimization.
```

File: _includes/common/ui/transactions-page.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+{% endif %}
+
+## Search and filter by application
+
+By default, this page shows transactions from all applications running on the cluster, and hides internal CockroachDB transactions.
+
+To filter the transactions by [`application_name`]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), use the **App** pulldown in the **Filters** menu. If you haven't set `application_name` in the client connection string, it appears as `unset`.
+
+- CockroachDB's internal transactions are only displayed under the `$ internal` app.
+- Transactions from the SQL shell are displayed under the `$ cockroach sql` app.
```
Comment (stbof) [inline _includes/common/ui/transactions-page.md:31]: You can also fitler by Query fingerprint runs longer than a time period. See the description in the Statement page.

File: _includes/common/ui/transactions-page.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+## Transaction statistics
+
+{% include common/ui/statistics.md %}
+
+For an example of querying the statistics table, see [Example]({{ page_prefix}}statements-page.html#example).
+
+## Filter by transaction latency
+
+You can filter transactions in which a SQL statement fingerprint exceeds a specified latency value. Use the pulldown in the **Filters** menu.
+
```
Comment (stbof) [inline _includes/common/ui/transactions-page.md:44]: I suggest moving this up to the Search section.

File: _includes/common/ui/transactions-page.md [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+  {% assign page_prefix = "ui-" %}
+{% endif %}
+
+The **Transactions** page helps you:
+
+- Identify frequently retried or high latency transactions.
+- View transaction [details](#transaction-details-page).
+
+{{site.data.alerts.callout_success}}
+In contrast with the [**Statements** page]({{ page_prefix }}statements-page.html), which displays [SQL statement fingerprints]({{ page_prefix }}statements-page.html#sql-statement-fingerprints), the **Transactions** page displays SQL statement fingerprints grouped by [transaction]({{ link_prefix }}transactions.html).
```
Comment (stbof) [inline _includes/common/ui/transactions-page.md:15]:
```suggestion
In contrast to the [**Statements** page]({{ page_prefix }}statements-page.html), which displays [SQL statement fingerprints]({{ page_prefix }}statements-page.html#sql-statement-fingerprints), the **Transactions** page displays SQL statement fingerprints grouped by [transaction]({{ link_prefix }}transactions.html).
```

File: _includes/common/ui/transactions-page.md [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+
+For an example of querying the statistics table, see [Example]({{ page_prefix}}statements-page.html#example).
+
+## Filter by transaction latency
+
+You can filter transactions in which a SQL statement fingerprint exceeds a specified latency value. Use the pulldown in the **Filters** menu.
+
+## Understand the Transactions page
+
+Use this page to identify transactions that you may want to [troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html), such as transactions that are experiencing high latencies, multiple [retries]({{ link_prefix }}transactions.html#transaction-retries), or execution failures.
```
Comment (stbof) [inline _includes/common/ui/transactions-page.md:47]:
```suggestion
Use the Transactions page to identify transactions that you may want to [troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html), such as transactions that are experiencing high latencies, multiple [retries]({{ link_prefix }}transactions.html#transaction-retries), or execution failures.
```

**Review by stbof — CHANGES_REQUESTED** [review]
See suggestions.

File: _includes/common/ui/statements-page.md [hunk 13]
```diff
@@ -0,0 +1,261 @@
...
+
+Use the Statements page to identify SQL statements that you want to [troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html). This might include statements that are experiencing high latencies, multiple [retries]({{ link_prefix }}transactions.html#transaction-retries), or execution failures. You can optionally create and retrieve [diagnostics](#diagnostics) for these statements.
+
+{{site.data.alerts.callout_success}}
+If you haven't yet executed any queries in the cluster as a user, this page will be blank.
+{{site.data.alerts.end}}
+
+{% if page.cloud == true %}
+<img src="{{ 'images/cockroachcloud/statements_page.png' | relative_url }}" alt="{{ site.data.products.db }} Console Statements Page" style="border:1px solid #eee;max-width:100%" />
+{% endif %}
```
Comment (stbof) [inline _includes/common/ui/statements-page.md:94]: I think you should remove or update this screenshot.  It has the old columns and Serverless doesn't support Diagnostics.

Screen Shot 2021-11-15 at 10 23 51 AM

This are the current columns:
Screen Shot 2021-11-15 at 10 22 49 AM

FWIW, I removed the screenshot from DB Console docs.

File: _includes/common/ui/statements-page.md [hunk 14]
```diff
@@ -0,0 +1,261 @@
...
+
+  - **Mean rows/bytes read** displays the mean average number of rows and bytes [read from the storage layer]({{ link_prefix }}architecture/life-of-a-distributed-transaction.html#reads-from-the-storage-layer) for statements with this fingerprint within the last hour .
+  - **Max memory usage** displays the maximum memory used by a statement with this fingerprint at any time during its execution within the last hour or specified time interval.
+  - **Network usage** displays the amount of [data transferred over the network]({{ link_prefix }}architecture/reads-and-writes-overview.html) (e.g., between regions and nodes) for statements with this fingerprint within the last hour. If this value is 0, the statement was executed on a single node.
+  - **Max scratch disk usage** displays the maximum amount of data [spilled to temporary storage on disk]({{ link_prefix }}vectorized-execution.html#disk-spilling-operations) while executing statements with this fingerprint within the last hour or specified time interval.
+
+**Statement details** displays information about the execution of the statement.
+
+
```
Comment (stbof) [inline _includes/common/ui/statements-page.md:147]:
```suggestion
```

Comment (stbof) [reply _includes/common/ui/statements-page.md:147]: nit: extra line.

File: _includes/common/ui/sessions-page.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+- If a session is active, the most recent SQL statement is displayed in the **Statement** column.
+- If a session is idle, **Transaction Duration**, **Statement Duration**, and **Statement** will display `N/A`.
+- To view [details of a session](#session-details), click **Session Duration**.
+
+{{site.data.alerts.callout_info}}
+An active session can have an open transaction that is not currently running SQL. In this case, the **Statement** and **Statement Duration** columns will display `N/A` and **Transaction Duration** will display a value. Transactions that are held open can cause [contention]({{ link_prefix }}performance-best-practices-overview.html#understanding-and-avoiding-transaction-contention).
+{{site.data.alerts.end}}
+
+{% if page.cloud != true %}
+<img src="{{ 'images/v21.2/ui-sessions-page.png' | relative_url }}" alt="DB Console Database Tables View" style="border:1px solid #eee;max-width:100%" />
```
Comment (stbof) [inline _includes/common/ui/sessions-page.md:26]: If we have Alt text it should be accurate. Right now it is the same in all screenshots.

**Review by stbof — APPROVED** [review]
Please remove the obsolete screenshot. Otherwise LGTM.

**Pull request merged**